frontend/packages/kubevirt-plugin/integration-tests/tests/vm.detail.node-selector.scenario.ts

```diff
@@ -1,17 +1,19 @@
 import { browser, ExpectedConditions as until } from 'protractor';
 import { testName } from '@console/internal-integration-tests/protractor.conf';
-import { createResource, deleteResource, click } from '@console/shared/src/test-utils/utils';
-import * as editNodeSelectorView from '../views/dialogs/editNodeSelectorView';
+import { createResource, deleteResource } from '@console/shared/src/test-utils/utils';
 import * as virtualMachineView from '../views/virtualMachine.view';
-import { saveButton } from '../views/kubevirtUIResource.view';
 import { VM_CREATE_AND_EDIT_TIMEOUT_SECS } from './utils/consts';
 import { VirtualMachine } from './models/virtualMachine';
 import { getVMManifest } from './utils/mocks';
 import { getRandStr } from './utils/utils';
+import { MatchLabels } from '@console/internal/module/k8s';
 
 describe('KubeVirt VM detail - edit Node Selector', () => {
   const testVM = getVMManifest('Container', testName, `node-selector-vm-${getRandStr(5)}`);
-  const vm = new VirtualMachine(testVM.metadata);
+  const vm: VirtualMachine = new VirtualMachine(testVM.metadata);
+  const labels: MatchLabels = {
```

Comment:
I would suggest to add at least two labels for this tests to make sure it works well. And then refactor `deleteLabel` to `deleteLabels`.

Reply:
updated, adding and removing 2 key:value labels, added `deleteNodeSelectors` method.

Reply:
Looks good.